### PR TITLE
Fix tutorials/qaoa.ipynb as part of docs cleanup for Cirq 1.0

### PR DESCRIPTION
**Pull request by tanujkhattar**
* Change example device from `cg.Bristlecone` (now deprecated) to `cg.Sycamore`.
* Use new GridDeviceMetadata instead of deprecated device properties (like neighbors_of)
* Move import to top
* Clear outputs

### CODE REVIEW

File: docs/tutorials/qaoa.ipynb [hunk 1]
```diff
@@ -1,24 +1,16 @@
 {
```
Comment (dabacon) [inline docs/tutorials/qaoa.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.    nx.set_edge_attributes(working_graph, {e: {'weight': np.random.randint(0, 500) / 100} for e in working_graph.edges})
</pre><p>I'd keep a version of the comment what this is doing as in the previous notebook, (i.e. "randomly weighted edge as arandom 2 decimal floating point between 0 and 5")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by dabacon — CHANGES_REQUESTED** [review]
One minor fix

File: docs/tutorials/qaoa.ipynb (hunk 1 above)
Comment (tanujkhattar) [reply docs/tutorials/qaoa.ipynb:1]: done

**Comment by tanujkhattar** [comment]
@dabacon Addressed the nit, this is ready for re-review.

File: docs/tutorials/qaoa.ipynb (hunk 1 above)
Comment (dabacon) [inline docs/tutorials/qaoa.ipynb:1]: <p>nittiest of nit nits: extra space after better when you removed the extra better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Review by dabacon — APPROVED** [review]

File: docs/tutorials/qaoa.ipynb [hunk 2]
```diff
@@ -623,12 +493,12 @@
     "id": "auo2VuTm6haO"
    },
    "source": [
-    "The QAOA is known to do just a little better better than random guessing for Max-Cut on 3-regular graphs at `p=1`. You can use very similar logic to the code above, but now instead of relying on the QAOA to decied your `S_partition` and `T_partition` you can just pick then randomly:"
+    "The QAOA is known to do just a little better than random guessing for Max-Cut on 3-regular graphs at `p=1`. You can use very similar logic to the code above, but now instead of relying on the QAOA to decide your `S_partition` and `T_partition` you can just pick then randomly:"
```
Comment (tanujkhattar) [inline docs/tutorials/qaoa.ipynb:496]: > nittiest of nit nits: extra space after better when you removed the extra better

@dabacon It looks like a reviewnb bug -- there's no extra space in text as seen here. Merging now.

**Pull request merged**